app/views/clients/_files.haml

```diff
@@ -0,0 +1,20 @@
+.col-md-6.c-card__content-group
+  .c-card__content-group-heading
+    .c-card__content-group-icon.icon-clipboard-check
+    %h5 Files
+  - if @files.present?
+    %table.table
+      %thead
+        %tr
+          %th File
+          %th Effective Date
```

Comment:
Hmm, is this effective date from ClientFile in the warehouse? I believe potentially there is also an expiration date, we should show both if that's what this is since some file tags require effective date, some require expiration date, some require neither.